src/arithmetic/algebraic_expression/algebraic_expression_add.c

```
@@ -56,22 +60,29 @@ Delta_Matrix _Eval_Add
+				: GrB_DESC_T1
+			: desc;
 	}
 
 	//--------------------------------------------------------------------------
 	// perform addition
 	//--------------------------------------------------------------------------
 
-	info = Delta_eWiseAdd(res, GxB_ANY_PAIR_BOOL, A, B);
+	info = Delta_eWiseAdd(res, GxB_ANY_PAIR_BOOL, A, B, desc);
 	ASSERT(info == GrB_SUCCESS);
 
```

cppcheck (2.10-2)

[error] Memory leak

(memleak)